### PR TITLE
feat(e2e): Add e2e-test suit for device-localpv

### PR DESCRIPTION
**Pull request by w3aman**
Signed-off-by: w3aman <aman.gupta@mayadata.io>

- This PR adds e2e-test suit for device-localpv

### CODE REVIEW

**Comment by w3aman** [comment]
cc: @nsathyaseelan for review

File: e2e-tests/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#Installing necessary ubuntu packages
+RUN rm -rf /var/lib/apt/lists/* && \
+    apt-get clean && \
+    apt-get update --fix-missing || true && \
+    apt-get install -y python python-pip netcat iproute2 jq sshpass bc git\
+    curl openssh-client
+
+#Installing ansible
+RUN pip install ansible==2.7.3
+
```
Comment (nsathyaseelan) [inline e2e-tests/Dockerfile:28]: @w3aman  can you add this dependent package before openshift package installation `RUN pip install ruamel.yaml.clib==0.1.2`

File: e2e-tests/apps/busybox/liveness/test.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  gather_facts: False
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+        - block:
+
+            - name: Record test instance/run ID
```
Comment (nsathyaseelan) [inline e2e-tests/apps/busybox/liveness/test.yml]: can we use the util already created for this task?

File: e2e-tests/apps/percona/workload/test.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+
+        - block:
+            - name: Record test instance/run ID
+              set_fact:
+                run_id: "{{ lookup('env','RUN_ID') }}"
```
Comment (nsathyaseelan) [inline e2e-tests/apps/percona/workload/test.yml]: Here also can we use the util?

**Review by nsathyaseelan — CHANGES_REQUESTED** [review]

**Review by nsathyaseelan — APPROVED** [review]
lgtm

**Pull request merged**